spec/views/users/_procedure_footer.html.haml_spec.rb

```diff
@@ -27,4 +27,47 @@
     it { is_expected.to have_link("Accessibilité") }
     it { is_expected.not_to have_text('téléphone') }
   end
+
+  describe '#cadre_juridique' do
+    context 'quand c\'est un lien texte' do
```

Comment:
Les descriptions de test sont en anglais d'habitude, non ?

```suggestion
    context 'when an external link is provided' do
```